### ISSUE TITLE
Support for Location List

### ISSUE THREAD

**Issue by Integralist** [ledesmablt/vim-run#3]
👋🏻 @ledesmablt as per the title is there any chance this plugin could support writing to a location list instead of a quick fix window? The reason I ask is because I primarily work with the quick fix window, as the majority of tools I have use the quick fix window, and have a tendency to overwrite what's in my quick fix window even when I don't want them to. But rarely do tools write output to the location list (with the exception of builtin tools like `lvimgrep`).

By allowing me to configure the location list as the destination, I can feel more confident in running this plugin multiple times and not having to worry about another LSP style tool steamrolling over the list of results.

Not the end of the world if this isn't something you think that would make sense to support. But I would definitely love it could be tweaked to work with a location list instead.

Oh and before I go I wanted to just say I **_really_** appreciate your work on this project. Its simplicity is a breath of fresh air compared to alternatives like `skywind3000/asyncrun` (along with `voldikss/vim-floaterm`). For both of said tools I found the user experience to be very rough.

The help documentation for this project in Vim is also excellent. Concise and relevant without blowing your mind with options and overloaded with features that aren't practical in the day-to-day. I scanned through the list of commands and found myself thinking "yup, useful, wonder if I can do... yup that's there too, yup yup ...cool that's everything! and my brain doesn't hurt" -- very straightforward and focused command set.

Thank you! 👍🏻 🙂

**Comment by ledesmablt** [comment]
@Integralist thank you so much for the support! This feature request is a great idea. I added the option `g:run_use_loclist` in [this commit](https://github.com/ledesmablt/vim-run/commit/1ab5e0156b6293ec1f7d3814cd5eaaed5240ac16) which should do the trick. Feel free to close this issue if you've tried it and it works fine.

On a sidenote, I also tried out those alternatives you mentioned + `tpope/vim-dispatch` (great plugin) but personally wanted something more intuitive. I'm very glad you appreciate the simplicity since that's what I was going for =)
